packages/subgraph/tests/utils/events/plugin.ts

```diff
@@ -1,22 +1,332 @@
-import {NumberStored} from '../../../generated/templates/Plugin/Plugin';
-import {Address, BigInt, ethereum} from '@graphprotocol/graph-ts';
-import {newMockEvent} from 'matchstick-as';
+import {MultisigPlugin, MultisigProposal} from '../../../generated/schema';
+import {
+  ProposalCreated,
+  Approved,
+  ProposalExecuted,
+  MembersAdded,
+  MembersRemoved,
+  MultisigSettingsUpdated,
+} from '../../../generated/templates/Plugin/Plugin';
+import {ADDRESS_ONE, DAO_ADDRESS, CONTRACT_ADDRESS} from '../constants';
+import {
+  generatePluginEntityId,
+  generateProposalEntityId,
+} from '@aragon/osx-commons-subgraph';
+import {Address, BigInt, Bytes, ethereum} from '@graphprotocol/graph-ts';
+import {createMockedFunction, newMockEvent} from 'matchstick-as';
 
-export function createNewNumberStoredEvent(
-  number: string,
+// test data
+
+export const ZERO = '0';
+export const ONE = '1';
+export const TWO = '2';
+export const THREE = '3';
```

Comment:
high value add programming right here

Reply:
Yes..., all of this must be cleaned up/refactored.